server/src/azrael/query_resolvers.py

```diff
@@ -0,0 +1,126 @@
+
+
+from typing import Callable, Dict, Optional
+
+import pymongo
+from db import DB
+from tartiflette import Resolver
+
+database_name = 'ethereum-indexer'
+collection_name = '0x94D8f036a0fbC216Bb532D33bDF6564157Af0cD7-state'
+
+db = DB()
+
+def _remove_event_field(doc): 
```

Comment:
let's make `doc` a dataclass, and all of these operations you have here like removing event field (could be done on dataclass instantiation), and casting will be all internal to it. Let's also give some more descriptive name other than having it as `doc`. There are a bunch of dataclass examples in this repo.

Reply:
done.